src/pages/events/details.astro

```diff
@@ -0,0 +1,425 @@
+              consequat. Aliquam erat volutpat. Praesent et nibh at tellus imperdiet consequat. Sed
+              felis metus, dapibus in gravida vehicula, pretium ut arcu. In condimentum nec magna non
+              faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos
+              himenaeos.
+            </p>
+          </div>
+        </div>
+    
+        <!-- event details content for location, attendance, dates, and cost -->
+        <div class="w-[352px] h-[576px] sm:order-2 order-1">
+          <p class="text-xl font-medium mb-4 text-gray-500">Event details</p>
+          <hr class="mb-4" />
+          <!-- Location -->
+          <div class="flex items-center">
+            <!-- using the icon component -->
+            <Icon svgPaths={[
+              "M21 10C21 17 12 23 12 23C12 23 3 17 3 10C3 7.61305 3.94821 5.32387 5.63604 3.63604C7.32387 1.94821 9.61305 1 12 1C14.3869 1 16.6761 1.94821 18.364 3.63604C20.0518 5.32387 21 7.61305 21 10Z",
+              "M12 13C13.6569 13 15 11.6569 15 10C15 8.34315 13.6569 7 12 7C10.3431 7 9 8.34315 9 10C9 11.6569 10.3431 13 12 13Z", ]}
+              color="#F04438"  width="24" height="24" bgColor="bg-[#Fef3f2]" rounded="full" />
+    
+            <div class="ml-2 leading-6">
+              <p class="text-xl font-bold text-gray-500">Location</p>
+              <p class="font-normal text-gray-500">124, Johnson street, Asokoro, Abuja</p>
+            </div>
+          </div>
+          <!-- Attendance -->
+          <div class="flex items-center mt-8"> 
+            <!-- using the icon component -->
+            <Icon svgPaths={[
+              "M23 20.9999V18.9999C22.9993 18.1136 22.7044 17.2527 22.1614 16.5522C21.6184 15.8517 20.8581 15.3515 20 15.1299",
+              "M17 21V19C17 17.9391 16.5786 16.9217 15.8284 16.1716C15.0783 15.4214 14.0609 15 13 15H5C3.93913 15 2.92172 15.4214 2.17157 16.1716C1.42143 16.9217 1 17.9391 1 19V21",
+              "M16 3.12988C16.8604 3.35018 17.623 3.85058 18.1676 4.55219C18.7122 5.2538 19.0078 6.11671 19.0078 7.00488C19.0078 7.89305 18.7122 8.75596 18.1676 9.45757C17.623 10.1592 16.8604 10.6596 16 10.8799",
+              "M9 11C11.2091 11 13 9.20914 13 7C13 4.79086 11.2091 3 9 3C6.79086 3 5 4.79086 5 7C5 9.20914 6.79086 11 9 11Z"]}
+              color="#06AED4" width="24" height="24" bgColor="bg-[#ECFDFF]" rounded="full" />
+    
+            <div class="ml-2 leading-6">
+              <p class="text-xl font-bold text-gray-500">Attendance</p>
+              <p class="font-normal text-gray-500">Physical + Online</p>
+            </div>
+          </div>
+          <!-- Date & Time -->
+          <div class="flex mt-8">
+          <!-- using the icon component -->
+            <Icon svgPaths={[
+              "M19 4H5C3.89543 4 3 4.89543 3 6V20C3 21.1046 3.89543 22 5 22H19C20.1046 22 21 21.1046 21 20V6C21 4.89543 20.1046 4 19 4Z",
```

Comment:
Why is the `Icon` looking so complex? IMO, I don't think we need `svgPaths, color,`

And the icons should have a default values for width and height so they can be reuseable